Specifications/Language/2_Statements/QuantumMemoryManagement.md

```diff
@@ -16,52 +16,67 @@ At the end of the block, the bound variables go out of scope and are no longer d
 
 Q# distinguishes between the allocation of "clean" qubits, meaning qubits that are unentangled and are not used by another part of the computation, and what is often referred to as "dirty" qubits, meaning qubits whose state is unknown and can even be entangled with other parts of the quantum processor's memory.
 
-## Using-Statement
+## Use-Statement
 
-Clean qubits are allocated by the `using`-statement.
-The statement consists of the keyword `using`, followed by an open parenthesis `(`, a binding, a close parenthesis `)`, and the statement block within which the qubits will be available.
+Clean qubits are allocated by the `use`-statement.
+The statement consists of the keyword `use` followed by a binding and an optional statement block.
+If a statement block is present, the qubits are only available within that block.
+Otherwise, the qubits are available until the end of the current scope.
 The binding follows the same pattern as `let` statements: either a single symbol or a tuple of symbols, followed by an equals sign `=`, and either a single tuple or a matching tuple of *initializers*.
 
 Initializers are available either for a single qubit, indicated as `Qubit()`, or an array of qubits, `Qubit[n]`, where `n` is an `Int` expression.
 For example,
 
 ```qsharp
-using (qubit = Qubit()) {
+use qubit = Qubit();
+// ...
+
+use (aux, register) = (Qubit(), Qubit[5]);
+// ...
+
+use qubit = Qubit() {
     // ...
 }
 
-using ((aux, register) = (Qubit(), Qubit[5])) {
+use (aux, register) = (Qubit(), Qubit[5]) {
     // ...
 }
 ```
 
 The qubits are guaranteed to be in a |0⟩ state upon allocation. They are released at the end of the scope and are required to either be in a |0⟩ state upon release, or to have been measured right beforehand. This requirement is not compiler-enforced, since this would require a symbolic evaluation that quickly gets prohibitively expensive. When executing on simulators, the requirement can be runtime enforced. On quantum processors, the requirement cannot be runtime enforced; an unmeasured qubit may be reset to |0⟩ via unitary transformation. Failing to do so will result in incorrect behavior. 
 
-The `using`-statement allocates the qubits from the quantum processor's free qubit heap, and returns them to the heap after the statement terminates.
+The `use`-statement allocates the qubits from the quantum processor's free qubit heap, and returns them to the heap no latter than the end of the scope in which the qubits are bound.
```

Comment:
Is "latter" opposed to "later" here correct?

Reply:
No, that's a typo, thanks!